### PR TITLE
NN-2881 Add CSRA last review date to profile header

### PR DESCRIPTION
**Pull request by jsrobertson**
image

### CODE REVIEW

File: backend/services/prisonerProfileService.js [hunk 1]
```diff
@@ -129,6 +130,9 @@ module.exports = ({
 
     const canViewSocLink = Boolean(isSocUser && socDetails)
 
+    const mostRecentAssessment =
+      hasLength(assessments) && assessments.sort((a, b) => b.assessmentDate.localeCompare(a.assessmentDate))[0]
```
Comment (Nimphal) [inline backend/services/prisonerProfileService.js:134]: I think you want a more specific assessment here, one with the `assessmentCode: 'CSR'` . Definitely only ones relating to CSR, as re-categorisation assessments will also be returned in this array.

Comment (Nimphal) [reply backend/services/prisonerProfileService.js:134]: This is what we do in the cell move header:
```const showCsraLink = assessments => {
  return assessments.some(assessment => assessment.assessmentCode === 'CSR' && assessment.assessmentComment)
}```

Comment (jsrobertson) [reply backend/services/prisonerProfileService.js:134]: Thank you, didn't realise that.

**Review by Nimphal — APPROVED** [review]
LGTM

**Pull request merged**